src/components/MultiHopTrade/hooks/useThorStreamingProgress/useThorStreamingProgress.tsx

```diff
@@ -0,0 +1,130 @@
+import type { ProgressProps } from '@chakra-ui/progress'
+import axios from 'axios'
+import { getConfig } from 'config'
+import { useEffect, useMemo, useRef, useState } from 'react'
+import { usePoll } from 'hooks/usePoll/usePoll'
+const POLL_INTERVAL_MILLISECONDS = 30_000 // 30 seconds
+
+type ThornodeStreamingSwapResponse = {
+  tx_id?: string
+  interval?: number
+  quantity?: number
+  count?: number
```

Comment:
These seem undefined for non-streaming swaps, can we perhaps use them as discriminator on whether or not to use the progress bar / "Loading" copy?

Waited a few minutes to ensure the Tx status is properly picked by the THOR node and still got the same response.

<img width="581" alt="image" src="https://github.com/shapeshift/web/assets/17035424/94188064-ea50-4c72-8df7-48c2ed98ec2b">

This might be an edge case, but may be nice to tackle regardless 🙏🏽

Reply:
In this case I erred on the side of safety and used the schema from the swagger docs to define them as optional or required, and it seems there are no required fields defined for this response.

https://thornode.ninerealms.com/thorchain/doc/
![image](https://github.com/shapeshift/web/assets/125113430/929e6bdb-8e28-4df3-91bf-089cc9905fe8)